tests/unit/test_network_connections.py

```diff
@@ -2528,10 +2527,7 @@ def test_convert_passwd_flags_nm(self):
 
 class TestSysUtils(unittest.TestCase):
     def test_link_read_permaddress(self):
-        # Manipulate PATH to use ethtool mock script to avoid hard dependency on
-        # ethtool
-        os.environ["PATH"] = TESTS_BASEDIR + "/helpers:" + os.environ["PATH"]
-        self.assertEqual(SysUtil._link_read_permaddress("lo"), "23:00:00:00:00:00")
+        self.assertEqual(SysUtil._link_read_permaddress("lo"), "00:00:00:00:00:00")
```

Comment:
Please add tests for a non-existing interface and an invalid interface (more than 16 characters), too.

Reply:
Done, thanks!